defaults.yml

```
@@ -171,6 +171,27 @@ categories:
             name: com.apple.dock autohide
             value: true
         after: killall Dock
+      - key: persistent-apps
+        domain: com.apple.dock
+        title: Add spacer tile
+        description: Appends a spacer tile to the list of apps in the dock.
+        param:
+          type: array-add
+        examples:
+          - value: "'{\"tile-type\"=\"spacer-tile\";}'"
+            image:
+              filename: "spacer-tile.png"
+              width: 740
+              height: 463
+            text: Appends a spacer tile to the persistent apps area of the dock.
+          - value: "'{\"tile-type\"=\"small-spacer-tile\";}'"
+            image:
+              filename: "small-spacer-tile.png"
+              width: 740
+              height: 463
+            text: Appends a spacer tile to the persistent apps area of the dock.
+        versions: [Big Sur, Catalina, Mojave]
```

```suggestion
        deleteAlert:
          type: danger
          message:
            "This command will remove all saved Apps from your Dock!<br>
            Make sure to save your config using<br>
            `defaults export com.apple.dock defaults-dock-backup.plist`<br>
            You can restore it using<br>
            `defaults import com.apple.dock defaults-dock-backup.plist`"
        versions: [Big Sur, Catalina, Mojave]
```

You should be able to use this now 🙂 

(See https://github.com/yannbertrand/macos-defaults/commit/71eb300dd6c62ca7fa78f8e1200c27b0824c6849 for more details on implementation)